modules/AOW_Actions/AOW_Action.php

```diff
@@ -106,7 +106,7 @@ public function save_lines($post_data, $parent, $key = '')
                             } else {
                                 if ($post_data[$key . 'param'][$i]['value_type'][$p_id] == 'Value' && is_array($p_value)) {
                                     $param_value[$p_id] = encodeMultienumValue($p_value);
-                                }else{
+                                }elseif($post_data[$key . 'param'][$i]['value_type'][$p_id] == 'Value'){
```

Comment:
Hi @ojs87 @jack7anderson7  could we be missing some scenarios by not including an else? (even if not calling fixUpFormatting)

Reply:
I didn't think it would be necessary, as the else was originally added specifically to target post_data of type "Value". The issue arose because it was also changing post_data of type "Field" which fixUpFormatting() is not designed for.

Reply:
Hi @ojs87, thank you